sdk/monitor/monitor-opentelemetry/src/metrics/performanceCounters.ts

```diff
@@ -0,0 +1,405 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as os from "os";
+import type {
+  Histogram,
+  Meter,
+  ObservableCallback,
+  ObservableGauge,
+  ObservableResult,
+} from "@opentelemetry/api";
+import { SpanKind, ValueType } from "@opentelemetry/api";
+import { AzureMonitorMetricExporter } from "@azure/monitor-opentelemetry-exporter";
+import type {
+  MeterProviderOptions,
+  PeriodicExportingMetricReaderOptions,
+} from "@opentelemetry/sdk-metrics";
+import { MeterProvider, PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics";
+import type { ReadableSpan, TimedEvent } from "@opentelemetry/sdk-trace-base";
+import { PerformanceCounterMetricNames } from "./types.js";
+import type { AzureMonitorOpenTelemetryOptions } from "../types.js";
+import { getLogData, isExceptionData } from "./quickpulse/utils.js";
+import type { ExceptionData, TraceData } from "./quickpulse/types.js";
+import { LogRecord } from "@opentelemetry/sdk-logs";
+import { Logger } from "../shared/logging/logger.js";
+
```

Comment:
We should import `node:process` here to ensure that we always have the node process